tests/src/Kernel/DataProducer/EntityTest.php

```diff
@@ -222,10 +222,14 @@ public function testResolveLabel(): void {
 
     $this->assertEquals('Dummy label', $this->executeDataProducer('entity_label', [
       'entity' => $this->entity,
+      // @todo Remove when https://github.com/drupal-graphql/graphql/issues/1233 is fixed.
+      'access' => TRUE,
```

Comment:
this should hopefully not be needed once we have proper default value handling above.

Reply:
Now that the issue is fixed, I removed the code.